client/src/components/common/button/login-button.tsx

```diff
@@ -0,0 +1,20 @@
+import { useNavigate } from "@tanstack/react-router";
+const LoginButton = () => {
+  const navigate = useNavigate();
+  return (
+    <div className="flex flex-col justify-center items-center">
+      <button
+      onClick={() => {
+        navigate({ to: "/home" });
+      }}
+        type="submit"
+        className="relative w-[152px] h-[27px]  top-[104px] border-radius
```

Comment:
tailwindでborder-radiusは[rounded](https://tailwindcss.com/docs/border-radius)だね
```suggestion
        className="relative w-[152px] h-[27px]  top-[104px] rounded-[4px]
```

Reply:
あとまあこれは修正しなくていいけど、今回ログインボタンの位置をpositionを使って相対配置で決めているけど、そんな必要はないね。
普通に、html上で上に書いてあるタグは下に書いてあるタグよりも画面の中で上に表示されるからね。あとはdisplay: flexとflex-direction: column, justify-content: centerを用いれば、中央揃えで縦並びにもできるはず。

positonは物を重ねる時以外はまず使わないので、今後は意識してみて